css/types/counter.json

```
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "types": {
+      "attr": {
```

Missed this on copy and paste?

```suggestion
      "counter": {
```